h/templates/activity/search.html.jinja2

```diff
@@ -122,7 +122,7 @@
     <div class="search-result-bucket__annotation-cards-container" data-ref="content">
       <ol class="search-result-bucket__annotation-cards">
         {% for result in bucket.presented_annotations %}
-          {{ annotation_card(result.annotation, request, result.group) }}
+          {{ annotation_card(result.annotation, request, result.incontext_link, result.group) }}
```

Comment:
If you do want to reduce the number of arguments here you could just pass `result` instead of the three `result.*`'s